api/src/main/java/org/jboss/forge/roaster/Roaster.java

```diff
@@ -93,36 +98,59 @@ private Roaster()
             return result;
          }
       }
-      throw new ParserException("Cannot find JavaParserProvider capable of producing JavaSource of type "
```

Comment:
This is still required for cases when roaster-jdt is not in classpath

Reply:
Valid point. Will change this